src/UraniumUI.Material/Controls/TreeViewNodeHolderView.cs

```diff
@@ -224,7 +204,64 @@ protected override void OnBindingContextChanged()
     {
         base.OnBindingContextChanged();
         OnSelectedItemChanged();
+
+        if (nodeChildren == null && ChildrenBinding is Binding binding &&
+        !string.IsNullOrEmpty(binding.Path) &&
+        TryGetInitialChildren(binding.Path, out var children) &&
+        (children?.Cast<object>().Any() ?? false))
```

Comment:
same as previous comment, merely `children.Cast<object>().Any()`

Reply:
Also:

- in both methods, nothing is done with the IEnumerable returned in `children`?
- what is here (logic) looks very duplicated to the code in lines 97-100. could this be consolidated?